felix/dataplane/linux/int_dataplane.go

```diff
@@ -837,10 +838,15 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 	// Register that we will report liveness and readiness.
 	if config.HealthAggregator != nil {
 		log.Info("Registering to report health.")
+		timeout := config.WatchdogTimeout
+		if timeout < healthInterval*2 {
+			log.Warnf("Dataplane watchdog timeout too low, defaulting to %v", healthInterval*2)
```

Comment:
nit: Would be nice to log the too-low value, in case user is confused about what they've configured.